app/src/main/java/com/kelvin/parkr/ui/DashboardFragment.kt

```diff
@@ -0,0 +1,82 @@
+package com.kelvin.parkr.ui
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProviders
+import com.kelvin.parkr.R
+import com.kelvin.parkr.ui.adapter.DashboardPagerAdapter
+import com.kelvin.parkr.viewModel.DashboardViewModel
+import kotlinx.android.synthetic.main.fragment_dashboard.*
+
+class DashboardFragment : BaseFragment() {
+    private lateinit var dashboardVm: DashboardViewModel
```

Comment:
could we name this `dashboardViewModel` instead? Since in software engineering it could also mean virtual memory.